src/js/content/content.spec.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2020 Knot.x Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { getData } from './content';
+
+
+describe('test ', () => {
+  beforeAll(async () => {
+    chrome.runtime.sendMessage = jest.fn(() => {});
+    global.fetch = jest.fn(() => Promise.resolve({
+      json: () => Promise.resolve({}),
+    }));
+    await getData('application/json');
+
+
+  });
+  it('test 1', () => new Promise(() => {
```

Comment:
this test fail, because Jest can't read response of chrome.runtime.sendMessage() from content.js file - function getData.